app/onepassword_events_api/appserver/static/javascript/views/app.js

```diff
@@ -67,68 +58,49 @@ export default class SetupPage extends React.Component {
   validateJWT(token) {
     const tokenComponents = token.split(".");
     if (tokenComponents.length !== 3) {
-      return "Invalid JSON Web Token - too short";
+      return "This doesn't look like a valid JSON Web Token.";
```

Comment:
Just a small thing, but if you want a constant message returned, I'd suggest just assigning it to a variable and returning that in each place.